index.html

```diff
@@ -109,10 +112,10 @@ <h2 class="grey_words">2015</h2>
                     <li><button class="lang_type">css</button></li>
                     <li><button class="lang_type">javaScript</button></li>
                 </ul>
-            <button class="white_button">See Project</button>
+            <button id="submitTwo" class="white_button">See Project</button>
             </div>
         </div>
-        <div class="project">
+        <div id="third-item" class="project">
             <img class="project_image" src="images/work3.png" alt="third project">
             <img class="project_image_desktop" src="images/project3_desktop.png" alt="third project">
             <div class="desktop_text">
```

Comment:
- [ ] Kindly try as much as you can to match the original design
     - Position your elements correctly. The language elements should be below the image and not above the image
     - Adjust the `border-radius` , `text-color` and also add the relevant icons to the `see live` and `see source` buttons
     - Kindly ensure that your `pop-up`  fits exactly on the viewport of every screen by giving it a height of `100 vh`
     
![image](https://user-images.githubusercontent.com/98453979/216072306-c3e3fa52-2559-4fbf-978b-845bd71ed384.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/216072507-c7803025-caba-4703-9f25-8fb727ceb3be.png)

Reply:
Position elements - Fixed 🔧 
Adjust borders, text and icons of buttons - Fixed 🔧 
Pop-up height to 100vh - Fixed 🔧

![image](https://user-images.githubusercontent.com/108192399/216105540-ffb5d74b-8b7c-4d06-9d03-f7edb34a8b38.png)

Reply:
- You seem to have introduced a bug as a result of the changes you made. It seems like the image takes a lot of the space so I cant view the description section and the links button. Please fix this 🙏 

Your project:
<img width="941" alt="eergwrg" src="https://user-images.githubusercontent.com/73607512/216144539-e125f661-4441-4976-9a0f-90bb54ec6f28.PNG">